### PR TITLE
[Modular] Contractor override

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
This pull request puts in some much needed freedom from Contractors for a bit, giving them a more sensible cost per contractor and reducing them to only one spawn per Contractor wave.

This would normally be done in Dynamic.json, but because for some reason this antag doesn't care about what the config says, it has to be done with a modular override. Womp womp

## Why It's Good For The Game
Makes contractor less common, and less god-awful to deal with. They spawn every 2 or 3 rounds with only slightly less frequency than traitors, at a lower cost currently.

## Changelog

:cl: ReturnToZender
balance: Only one contractor spawns now, and its cost is appropriately set
/:cl:

### CODE REVIEW

File: modular_zubbers/code/datums/midround/ruleset.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/datum/dynamic_ruleset/midround/from_ghosts/contractor
+	required_candidates = 1
+	weight = 4
+	cost = 14
```
[the remark this answers is not in the document]
Comment (ReturnToZender) [reply modular_zubbers/code/datums/midround/ruleset.dm:4]: True but cost isn't proportional to the threat, nukeops have a cost of 7, which is why blob isn't normally high cost, but is simply low weight

cost of these midrounds generally hovers around 10-14 (think Space Ninja and Ling) though I kinda overshot it a bit

[the remark this answers is not in the document]
Comment (ReturnToZender) [reply modular_zubbers/code/datums/midround/ruleset.dm:4]: Also note that on the config repo Blob costs 15 right now

[the remark this answers is not in the document]
Comment (projectkepler-ru) [reply modular_zubbers/code/datums/midround/ruleset.dm]: > blob costs eight
> 
> i don't think one contractor is worth two blobs

I beg to differ

[the remark this answers is not in the document]
Comment (radarseas) [reply modular_zubbers/code/datums/midround/ruleset.dm]: space ninja and changling have infinite EMPs. that can absolutely firelock hell a station though which justifies their cost. plus if both know what they're doing, they can enter public spaces and do this very easily and be nearly impossible to contain.

Not to mention ninjas have a very low skill requirement and in the case of changling, you can very quickly disappear out of sight and you get free armblade + spaceproofing on top of your gear to mix with armors you can't normally wear in space, _think ablative trenchcoat._ and you get the idea, they are more of a problem, more consistantly for everyone involved.

Contractor is a hit and miss, I'd put it at... 9? 8? they're not always guarrented good starting items either, or good conditions to arrive to, a paddy mech practically hardcounters contractors nine times out of ten.

[the remark this answers is not in the document]
Comment (radarseas) [reply modular_zubbers/code/datums/midround/ruleset.dm]: also, ninjas have no ID and infinite doorjack gloves! which encourages them to perma-open everything they touch. 

spacelings are encouraged to gank someone and replace them to infiltrate the station so they can start using maintenance to evade security patrols and sneak/break into places for stuff they might want.

**Comment by ReturnToZender** [comment]
Due to changes to Contractors, this PR is no longer necessary, to my knowledge

**Pull request closed without merging**